### PR TITLE
Switch to Jinja2 as templating engine.

### PR DESCRIPTION
**Pull request by icemac**
Previously we used %-formating.
I kept `%(` and `)s` as start resp. end of variables as the defaults of Jinja2 interfere with the format used in the GitHub actions configuration file.

### CODE REVIEW

File: config/README.rst [hunk 1]
```diff
@@ -45,7 +46,7 @@ Each directory contains the following files:
 
   - This file is copied to `.gitignore`.
 
-* MANIFEST.in
+* MANIFEST.in.jj2
```
Comment (mgedmin) [inline config/README.rst:49]: This is the first time I see `.jj2`.  I've seen `.j2` before, for Jinja templates.

**Review by mgedmin — APPROVED** [review]
LGTM

The %(...)s thing takes some getting used to, but I understand the rationale.

File: config/README.rst (hunk 1 above)
Comment (icemac) [reply config/README.rst:49]: I am going to fix it. I've rarely used Jinja2 templates before.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😀

**Pull request merged**